src/ui/Icon/Icon.ts

```diff
@@ -43,32 +39,36 @@ export class Icon extends Component {
     /**
      * Specifies the value that the Icon component should output as its CSS class instead of the auto-selected value.
      *
-     * Default value is `undefined`, and the framework will find a suitable icon based on the result file type.
+     * Default value is `undefined`, which means that the Coveo JavaScript Search Framework outputs a suitable icon
+     * depending on the result file type.
      */
     value: ComponentOptions.buildIconOption(),
 
     /**
-     * Specifies whether the Icon component should output the smaller version of the icon instead of of the regular one.
+     * Specifies whether the Icon component should output the smaller version of the icon instead of the regular one.
      *
      * Default value is `false`.
      */
     small: ComponentOptions.buildBooleanOption(),
```

Comment:
No. We should remove the misleading comment though.

Default value is `undefined`